### PR TITLE
Remove tautologies

### PR DESCRIPTION
**Pull request by greg0ire**
I am targeting this branch, because this is pedantic.

This removes a lot of comments that are just tautological. Also, this migrates more classes to Twig namespaces. I ran my script only on `src/Twig` the other day 😅

### CODE REVIEW

File: src/Formatter/Pool.php [hunk 1]
```diff
@@ -14,9 +14,9 @@
 use Psr\Log\LoggerAwareInterface;
 use Psr\Log\LoggerInterface;
 use Psr\Log\NullLogger;
-use Twig_Environment;
-use Twig_Error_Syntax;
-use Twig_Sandbox_SecurityError;
+use Twig\Environment;
+use Twig\Error\SyntaxError as TwigSyntaxError;
```
Comment (core23) [inline src/Formatter/Pool.php]: Why did you use an alias?

Comment (greg0ire) [reply src/Formatter/Pool.php]: Hmmm I don't know why rector did this, let's remove it.

**Comment by greg0ire** [comment]
ping @core23

**Comment by greg0ire** [comment]
ping @core23

File: src/Form/Type/SimpleFormatterType.php [hunk 2]
```diff
@@ -39,20 +38,20 @@ class SimpleFormatterType extends AbstractType
     private $templateManager;
 
     /**
-     * @param ConfigManagerInterface   $configManager   An Ivory CKEditor bundle configuration manager
-     * @param PluginManagerInterface   $pluginManager   An Ivory CKEditor bundle plugin manager
-     * @param TemplateManagerInterface $templateManager An Ivory CKEditor bundle template manager
+     * @param ConfigManagerInterface    $configManager   An Ivory CKEditor bundle configuration manager
+     * @param ?PluginManagerInterface   $pluginManager   An Ivory CKEditor bundle plugin manager
```
Comment (core23) [inline src/Form/Type/SimpleFormatterType.php]: This is wrong. Should be:

```
PluginManagerInterface|null   $pluginManager   An Ivory CKEditor bundle plugin manager
```

Comment (greg0ire) [reply src/Form/Type/SimpleFormatterType.php]: Are you sure this is wrong? See https://github.com/phpDocumentor/TypeResolver/pull/35/files#diff-58912ee2ce6f273d72422833d8e0721fR48

Comment (core23) [reply src/Form/Type/SimpleFormatterType.php]: Never seen this before... What style is used inside symfony code?

**Comment by greg0ire** [comment]
@core23 I reverted to `|null` because apparently people don't do that (yet, I asked), and the codebase isn't 7.1 only yet. Please review again.

**Pull request merged**